command/oauth/cmd.go

```diff
@@ -902,7 +940,7 @@ func (o *oauth) Auth() (string, error) {
 	if o.loginHint != "" {
 		q.Add("login_hint", o.loginHint)
 	}
-	u.RawQuery = q.Encode()
+	u.RawQuery = fmt.Sprintf("%s&%s", q.Encode(), o.authParams.Encode())
```

Comment:
I would say is cleaner to merge those two url.Values rather than appending them like that.